securedrop/crypto_util.py

```diff
@@ -123,23 +102,6 @@ def do_runtime_tests(self) -> None:
         if not rm.check_secure_delete_capability():
             raise AssertionError("Secure file deletion is not possible.")
 
-    def display_id(self) -> str:
-        """Generate random journalist_designation until we get an unused one"""
-
-        tries = 0
-
-        while tries < 50:
-            new_designation = ' '.join([random.choice(self.adjectives),
-                                        random.choice(self.nouns)])
-
-            collisions = Source.query.filter(Source.journalist_designation == new_designation)
```

Comment:
This was a bit problematic because it pulls a database session "out of thin air" (by doing `Source.query()`) instead of `database_session.query()`), so `display_id()` could only work when called from the correct context, which can be complicated.